### PR TITLE
add libvirt xml for Mojave

### PR DESCRIPTION
**Pull request by xuhuisheng**
add macOS-Mojave-libvirt.xml for Mojave. just copy from macOS-HS-libvirt.xml, change the related text in xml, the configuration is all the same.
Test in Ubuntu 18.10.

### CODE REVIEW

**Comment by kholia** [comment]
Thanks.

Instead of adding a similar file (with cosmetic changes), can we make the `macOS-HS-libvirt.xml` generic enough? Can `macOS-HS-libvirt.xml` be renamed to `macOS-libvirt.xml`.

Less duplication -> less maintenance.

**Comment by xuhuisheng** [comment]
I just jump mojave, didn't check other versions, The HS and Mojave is exactly the same.

File: macOS-Mojave-libvirt.xml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+-->
+  <name>macos-mojave</name>
+  <uuid>2aca0dd6-cec9-4717-9ab2-0b7b13d111c3</uuid>
+  <title>macOS Mojave</title>
+  <memory unit='KiB'>4194304</memory>
+  <currentMemory unit='KiB'>4194304</currentMemory>
+  <vcpu placement='static'>4</vcpu>
+  <os>
+    <type arch='x86_64' machine='pc-q35-2.11'>hvm</type>
+    <loader readonly='yes' type='pflash'>/home/CHANGEME/macOS/OVMF_CODE.fd</loader>
```
Comment (Jamesxu182) [inline macOS-Mojave-libvirt.xml]: It would be better to use a relative path instead of absolute path regarding OVMF

Comment (xuhuisheng) [reply macOS-Mojave-libvirt.xml]: libvirt seems only support absolute path.

**Comment by ShayBox** [comment]
Have you gotten this xml to work? I've gotten the qemu script to work, but the xml here freezes on the apple logo when starting the installer on Clover, I have to do more testing.

**Comment by xuhuisheng** [comment]
> Have you gotten this xml to work? I've gotten the qemu script to work, but the xml here freezes on the apple logo when starting the installer on Clover, I have to do more testing.

Test on Ubuntu 18.10. QEMU emulator version 2.12.0 (Debian 1:2.12+dfsg-3ubuntu8.3)

**Comment by kholia** [comment]
There is a single `macOS-libvirt.xml` file in the repository now.

**Pull request closed without merging**